hera_mc/sys_handling.py

```diff
@@ -369,3 +365,20 @@ def get_pam_info(self, stn, at_date, fc=None):
         H = cm_hookup.Hookup(self.session)
         pams = H.get_pam_from_hookup(hu)
         return pams
+
+    def publish_summary(self, hlist=['HH'], rev='A', exact_match=False,
+                        hookup_cols=['station', 'front-end', 'cable-post-amp(in)', 'post-amp', 'cable-container', 'f-engine', 'level']):
+        file2write = 'sys_conn.html'
+        location_on_paper1 = 'paper1:/home/davidm/local/src/rails-paper/public'
+        hookup_dict = self.hookup.get_hookup(hpn_list=hlist, rev=rev, port_query='all',
+                                             at_date='now', exact_match=exact_match, show_levels=True)
+        self.hookup.show_hookup(hookup_dict, hookup_cols, True, False, False, write_to_file=file2write)
```

Comment:
Once the arguments to `show_hookup` are fixed up, this chunk of code can become something like:

```
with open(file_to_write, 'w') as f:
  self.hookup.show_hookup(..., file=f, format='html')
```

The `with` statement makes sure that the file is closed, and gives precise control over when exactly that happens.